### DISCUSSION TITLE
Unity

### DISCUSSION THREAD

**Discussion by Arbee4ever** [catppuccin/catppuccin#1994] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
what are we supposed to do with this? 🤔

**Reply by Arbee4ever** [reply]
I tried searching for something to properly make Themes, but couldn't really find something, so I just posted this here, hoping someone could find something and maybe do it

**Reply by Arbee4ever** [reply]
nvm I found *something*: https://forum.unity.com/threads/custom-editor-theme.989456/

**Reply by GittyMac** [reply]
There is a way to theme the Unity Editor via the UI system's Unity Style Sheets (USS), but it's fairly under-documented for the editor. An Asset Store plugin exists for some simple tweaks, but there's not much documentation on how to theme certain unthemed elements, most notably the buttons. You can make a *mostly* complete theme, but there's some unthemed parts that stick out.

**Comment by FormalSnake** [comment]
I really need this

**Comment by BANanaD3V** [comment]
I can try to do it if it's unity editor but its gonna be long and tricky and painful cuz unity theming is bad. I also need this. If someone is willing to help my discord is @bananad3v
